app/views/smart_proxies/plugins/_puppet_ca.html.erb

```diff
@@ -0,0 +1,27 @@
+<h3><%= _("Puppet Certificate Authority") %></h3>
+<%= show_feature_version('puppetca') %>
```

Comment:
can you please show when the CA expires ? (the CA is the first certificate - oldest)
we can probably highlight (maybe in a separate PR or not) certs that will expire - but IMHO the CA cert will always expire first (and revoke all the other certs along) so other certificate expiry does not really matter.